tools/wasme/cli/pkg/deploy/istio/istio_provider.go

```diff
@@ -30,8 +30,9 @@ import (
 )
 
 const (
-	WorkloadTypeDeployment = "deployment"
-	WorkloadTypeDaemonSet  = "daemonset"
+	WorkloadTypeDeployment  = "deployment"
+	WorkloadTypeDaemonSet   = "daemonset"
+	WorkloadTypeStatefulSet = "statefulset"
```

Comment:
we'll want to update:
* https://github.com/solo-io/wasm/blob/master/tools/wasme/cli/operator/api/wasme/v1/filter_deployment.proto#L84
* https://github.com/solo-io/wasm/blob/master/tools/wasme/cli/pkg/deploy/istio/istio_provider.go#L368
* and https://github.com/solo-io/wasm/blob/master/tools/wasme/cli/pkg/cmd/deploy/options.go#L95

to reflect the new workload type. it might be helpful to encapsulate these in a `SupportedWorkloads = []string{...}` variable here which can be consumed in those places, so we don't have to manually update them in the future